recipes/r-rocs/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export DISABLE_AUTOBREW=1
+
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
+mkdir -p ~/.R
+echo -e "CC=$CC
```

Comment:
Are all of these needed to build?

Reply:
All of the r-rgl dependant packages that I've seen do this so I stuck with this by default. The package actually seem to run without *makevars* declaration, so it may be redundant.

Reply:
No harm in keeping it, thanks for checking!